migrate/livesync-for-s3.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Livesync S3 to Timescale Cloud
+excerpt: Synchronize data from S3 to Timescale Cloud service in real time
+products: [cloud]
+keywords: [migration, low-downtime, backup]
+tags: [recovery, logical backup, replication]
+---
+
+import PrereqCloud from "versionContent/_partials/_prereqs-cloud-only.mdx";
+import EarlyAccessNoRelease from "versionContent/_partials/_early_access.mdx";
+
+# Livesync from S3 to Timescale Cloud
+
+You use $LIVESYNC to synchronize all the data, or specific tables, from an S3 bucket to your
+$SERVICE_LONG in real-time. You run $LIVESYNC continuously, turning S3 into a primary database with your
+$SERVICE_LONG as a logical replica. This enables you to leverage $CLOUD_LONG’s real-time analytics capabilities on
+your replica data.
+
+![$LIVESYNC_CAP view status](https://assets.timescale.com/docs/images/livesync-s3-view-status.png)
+
+You use $LIVESYNC for data synchronization, rather than migration. Livesync can:
+
+* Sync data from an S3 bucket instance to a $SERVICE_LONG:
+   - $LIVESYNC uses Glob patterns to identify the objects to sync.
+   - $LIVESYNC uses the objects returned for subsequent queries. This efficient approach means files are synced in
+    [lexicographical order][lex-order].
+   - $LIVESYNC watches an S3 bucket for new files and imports them automatically. $LIVESYNC runs on a configurable 
+     schedule and tracks processed files.
+   - To prevent system overload, $LIVESYNC track up to 100 files for each sync iteration. Additional checks only fill 
+      empty queue slots.
+   - For large backlogs, $LIVESYNC checks every minute until caught up. 
+
+* Sync data from multiple file formats:
+
+  * CSV: checked for compression, then processing using [timescaledb-parallel-copy][parallel-copy]
+
+  * Parquet: converted to CSV, then processed using [timescaledb-parallel-copy][parallel-copy]
+
+* Enable features such as [hypertables][about-hypertables], [columnstore][compression], and
+  [continuous aggregates][caggs] on your logical replica.
+
+$LIVESYNC for S3 continuously imports data from an Amazon S3 bucket into your database. It monitors your S3 bucket for new
+files matching a specified pattern and automatically imports them into your designated database table.
+
+<EarlyAccessNoRelease />: livesync is not supported for production use. If you have an questions or feedback, talk to us in <a href="https://app.slack.com/client/T4GT3N2JK/C086NU9EZ88">#livesync in Timescale Community</a>.
+
+## Prerequisites
+
+<PrereqCloud />
+
+- Access to a standard Amazon S3 bucket containing your data files.
+  Directory buckets are not supported.
+- Access credentials for the S3 bucket. 
+  - The user needs the following permissions: `s3:GetObject`, `s3:ListBucket`.
+  - The following credentials are supported: 
+    - [IAM Role][credentials-iam].
```

Comment:
This must link to specific instructions. Users need to know which role they need to authorize. 

This is already in our UI, Maybe we should have a dedicated page in the documentation as well?

Reply:
I think having the link to these instructions in the UI and a link to the docs is enough. We only rewrite things for other products when the docs are terrible.

Reply:
This is not about rewriting docs, There is a process specific for our setup that it is not in AWS docs. 

I'm talking about this instructions in the UI
![image](https://github.com/user-attachments/assets/94db87eb-203e-4c07-adc2-e78ad51315bb)

user needs to know 
- The role we use is `"arn:aws:iam::142548018081:role/timescale-s3-connections"`
- We need permissions to `"s3:GetObject","s3:ListBucket"`
- We will send ExternalID with `ProjectID/ServiceID`

This is not available in AWS docs because it is specific for our setup